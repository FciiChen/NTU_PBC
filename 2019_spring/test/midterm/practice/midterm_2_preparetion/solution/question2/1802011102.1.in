QAU3z0, 3XHQ,RGDnnu
5BLDH, okyQ,rNjY0A9yFh
BREAK
QAU3z0 ,  Mlgv9b
5BLDH ,  KQWgOyR
BREAK
