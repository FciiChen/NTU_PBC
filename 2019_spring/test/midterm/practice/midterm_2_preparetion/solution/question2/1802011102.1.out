QAU3z0     3XHQ       RGDnnu     Mlgv9b    
5BLDH      okyQ       rNjY0A9yFh KQWgOyR   
